bin/update-package-references

```diff
@@ -44,43 +49,80 @@ function mkArtifactSha() {
     cp -r $DIR/../.git ./
     git checkout --orphan "$artifactName-$SHA"
     cp -r $orig/$targetDir/* ./
+    rm -rf packages || true
```

Comment:
When creating an artifact, the size of the resulting commit can be quite large if it also always includes the inner packages directory.  Removing this here is a simple reduction in artifact creation.